src/uu/tail/src/args.rs

```diff
@@ -479,6 +540,20 @@ pub fn uu_app() -> Command {
                 .num_args(1..)
                 .value_hint(clap::ValueHint::FilePath),
         )
+        .arg(
```

Comment:
I'm not sure about doing this with clap. These characters seem to be part of the obsolete syntax right? Couldn't we just fix the parsing for that and get it to work without additional flags? If not, then this is fine, but I think it'd be good to have a bit more comments to explain what these flags are for and how this parsing works.

Reply:
Yes, seems to be pretty obsolete.

The problem is, that calling `tail -l <file>` should behave like `tail -10l <file>`, which should basically behave like `tail -n 10 <file>` (`tail -b <file>` is similar). Basically, the flags are introduced only for these two corner cases.

Otherwise, clap is "stealing" them, and I am not sure how to work around it:
```
error: Found argument '-l' which wasn't expected, or isn't valid in this context

  If you tried to supply '-l' as a value rather than a flag, use '-- -l'
```

Reply:
So I can add a comment like "/* required for corner case: tail -l <file>*/", if wanted

Reply:
Ah sorry, I think I was being too vague. With obsolete syntax I mean the `-10` thing, which is parsed by this function: https://github.com/uutils/coreutils/blob/816e0d51e35697104f59427f4a85e77ae5f2c6fd/src/uu/tail/src/parse.rs#L15

The `l` and `c` are part of that argument, instead of being separate arguments, so it seems that we just have to fix this function to support `l` and `c`. We call this function before clap, so that works around clap stealing them :)

Reply:
Thanks for the hint, I guess the new location of the implementation fits better